data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,980 @@
+	autosave
+	landing
+	source "Hai-home"
+	clearance
+	to offer
+		has "Hai Leaks Response 4: done"
+	on offer
+		event "secret mountaintop"
+		event "role-play fleet deception"
+		event "slightly lost" 10
+		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
+		"reputation: Hai" >?= 100
+		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. Asked by Sayari to look into a rumor about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist.`
+		conversation
+			`When you return to Hai-home, you are met by a group of relieved diplomats. Of all of them, Alondo seems to be the least tired and takes the time to let you know what's been going on. "Captain <last>! You've missed quite the palaver. Still, I doubt you lost much. We've spent four hours trying to explain to Sayari what a 'furry' is."`
+			choice
+				`	"What? That could've been entertaining."`
+				`	"Hmm. No, I probably didn't miss much."`
+				`	"Oh, I could have told her all about it..."`
+			`	He chuckles. "Anyway, I have a hunch that we'll need you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
+			`	"I hear you've been instrumental in driving off some opportunistic pirates and helping stem the information leak. Come on, I'll buy you a drink on the others' behalf."`
+			`	Some hours later, you leave the fancy cocktail bar Alondo took you to and begin making your way back to your ship to sleep. Your cabin by far seems to be the best place to not be bothered.`
+			`	Partway there, a hooded figure suddenly falls in step beside you.`
+			choice
+				`	(Keep walking.)`
+				`	(Ask them who they are.)`
+			`	"Sssshhhh!" says the voice at your elbow. "Do not react, and do not be alarmed, it is merely I."`
+			`	You look at the figure out of the corner of your eye and connect the whispered voice to the squirrelly muzzle poking out. To your surprise, it is Sayari.`
+			`	"I should not be coming to you with this, so you must promise to say nothing of it to anyone just yet," she says.`
+			choice
+				`	"Of course."`
+			`	"A very sweet friend of mine, who is much too enthusiastic and perhaps led by the vagaries of youth but nonetheless quite astute, keeps harping on to me about this theory of hers that a smuggling ring is kidnapping Hai and transporting them into human space."`
+			`	"Of course, a thorough investigation has occurred on our side, and no evidence of such things has turned up, but..."`
```

Comment:
How can a thorough investigation turn up no evidence of people disappearing???
How deep does this smuggling ring go that it can remove people with such discretion?

(This seems a lot more like the kidnapping story I was promised.)